localstack/services/awslambda/invocation/plugins.py

```diff
@@ -0,0 +1,12 @@
+from localstack.services.awslambda.invocation.runtime_executor import RuntimeExecutorPlugin
```

Comment:
we're still importing this entire import path when we start localstack. which includes the lambda API, and everything under `localstack.services.awslambda.invocation.lambda_models`.
could we just move the `RuntimeExecutorPlugin` class here as well, to make sure we have no other imports?

Reply:
Ah, you are of course right. My assumption as that runtime_executor would not import anything as well, but of course it does for the types.